src/main/java/no/bibsys/handler/CodePipelineFunctionHandler.java

```diff
@@ -0,0 +1,74 @@
+package no.bibsys.handler;
+
+import com.amazonaws.services.codepipeline.AWSCodePipeline;
+import com.amazonaws.services.codepipeline.AWSCodePipelineClientBuilder;
+import com.amazonaws.services.codepipeline.model.ExecutionDetails;
+import com.amazonaws.services.codepipeline.model.FailureDetails;
+import com.amazonaws.services.codepipeline.model.FailureType;
+import com.amazonaws.services.codepipeline.model.PutJobFailureResultRequest;
+import com.amazonaws.services.codepipeline.model.PutJobSuccessResultRequest;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStreamWriter;
+import java.util.Optional;
+import no.bibsys.handler.requests.CodePipelineEvent;
+import no.bibsys.utils.IoUtils;
+
+public abstract class CodePipelineFunctionHandler<O> extends HandlerTemplate<CodePipelineEvent,O> {
+
+    private final transient AWSCodePipeline pipeline=AWSCodePipelineClientBuilder.defaultClient();
+
+    public CodePipelineFunctionHandler() {
+        super(CodePipelineEvent.class);
+    }
+
+    @Override
+    protected final CodePipelineEvent parseInput(InputStream inputStream) throws IOException {
+        String jsonSting= IoUtils.streamToString(inputStream);
+        System.out.println(jsonSting);
```

Comment:
We should introduce proper logging to CloudWatch over print statements

Reply:
I agree. I tried to use logging but I was doing something wrong and I could not get logs. 
We should create a task for logging